### ISSUE TITLE
Toggle Playback

### ISSUE THREAD

**Issue by Myztillx** [d8ahazard/Phlex#400]
Are there any plans for a toggle playback command? Such as: if playing, then pause and vice versa. 

I know from a spoken word point of view, this seems pointless; but I am trying to set up an automation with a button to simulate the play and pause. With a toggle command I would not need two buttons to produce the same result.

**Comment by d8ahazard** [comment]
If you look at the UI code for Flex TV, you'll see that I'm doing this by
tracking the playback state, and then displaying the appropriate play/pause
command in the UI. ;)

**Comment by Myztillx** [comment]
So this might be just for my "very" specific use case, but this is what I was wanting to use it for: 

I have set up a RFID reader to scan a card which will play a specific movie depending on which RFID is scanned. What I was trying to add is the ability to scan that same card again and toggle play/pause playback. 

I currently am working around this by making a dummy variable that stores this "assumed" state. I have set this up in node-red as pictured below.

The http requests details look like:
 https://app.phlexchat.com//api.php?say&apiToken=[token]&command={{payload}}
where the {{payload}} is set to either play or pause. 

What I was wondering is there a command I can send as {{payload}} that toggles these so the state is not assumed by the flow but is actual playback. Or if I can poll something for the current playback status.

This issue becomes a problem if someone changes the playback with something other than the card, such as a google home.

![image](https://user-images.githubusercontent.com/33730898/46092033-f3cc6200-c181-11e8-8574-4b2eca0b54c3.png)
